client/src/components/about1/aboutPage.css

```diff
@@ -0,0 +1,5 @@
+@import url("https://fonts.googleapis.com/css2?family=Anton&family=Inclusive+Sans:ital@0;1&family=Merriweather:ital,wght@0,300;0,400;0,700;0,900;1,300;1,400;1,700;1,900&family=Monomaniac+One&family=Mooli&family=Quicksand:wght@300&family=Roboto+Slab:wght@300&family=Roboto:ital,wght@0,300;1,400&display=swap");
+
+h1 {
+  font-family: "Merriweather", serif;
+}
```

Comment:
For some reason, the changes are overwriting the css of the sidebar
The sidebar's font code should be the same

![image](https://github.com/Abubakar-Meigag/FinalProject-DejaReview-teamUnity-Ldn10/assets/113212730/ecb5ea26-2547-4c20-9858-3bc0e8fce8ac)